Automattic/wp-calypso
# Update EditorCheckoutModal to use ShoppingCartProvider

#### Changes proposed in this Pull Request

* Update the `EditorCheckoutModal` component to use the `ShoppingCartProvider` which is needed as the sub-components within this checkout modal make use of `useShoppingCart` which need the provider.

#### Testing instructions

* Select a site that is on the free plan
* Open the editor on this site via a page/post
* Open DevTools > Console.
* Add a Premium block (e.g. Premium Content)
* Click "Upgrade" banner on the block
* A modal should appear over the post and the below error should *not* be present in your console.
* You should be able to edit/update your order and check out successfully via any of the available methods before being returned back into your post.

<img width="471" alt="Screenshot 2020-10-28 at 10 11 13" src="https://user-images.githubusercontent.com/8639742/97446543-6fcba200-1926-11eb-84de-3ceaaa474a88.png">
